velocity-transition-group.js

```diff
@@ -44,6 +44,13 @@ var ReactDOM = require('react-dom');
 var ReactTransitionGroup = require('react-addons-transition-group');
 var Velocity = require('./lib/velocity-animate-shim');
 
+// Shim requestAnimationFrame for browsers that don't support it, in particular IE 9.
+var shimRequestAnimationFrame = 
```

Comment:
You'll need to protect against `window` not being defined up here, since this code will execute regardless of whether it's browser or server environment. (The `Velocity.velocityReactServerShim` check inside of `render` prevented Node execution from ever getting to the previous `window` code.)